org.elysium.test/META-INF/MANIFEST.MF

```diff
@@ -8,4 +8,19 @@ Require-Bundle: org.eclipse.xtext;bundle-version="1.0.1",
  org.eclipse.xtext.junit;bundle-version="1.0.1",
  org.elysium;bundle-version="0.1.0",
  org.junit,
- javax.util;bundle-version="0.0.0"
+ javax.util;bundle-version="0.0.0",
+ org.elysium.ui,
+ org.eclipse.core.runtime,
+ org.eclipse.xtext.junit4,
+ org.eclipse.ui.workbench;resolution:=optional,
+ com.google.guava,
+ org.eclipse.xtext.xbase.lib,
+ org.eclipse.xtend.lib,
+ org.eclipse.xtend.lib.macro
+Export-Package: org.elysium
```

Comment:
There is a Plug-in Problem here on my machine: Package 'org.elysium' does not exist in this plug-in